x-pack/plugins/security_solution/public/resolver/view/panels/node_events_of_type.tsx

```diff
@@ -124,7 +123,8 @@ const NodeEventsListItem = memo(function ({
     panelParameters: {
       nodeID,
       eventCategory,
-      eventID: String(eventModel.eventID(event)),
+      eventID: eventID ? String(eventID) : '',
```

Comment:
Please don't use `''`.

Reply:
👍 , we did this because this value is used for the url params logic, stateSyncingActions I believe. Took us a while to catch that we were getting 'undefined' rather than the actual undefined value.

Reply:
@jonathan-buttner fixed